File: PythonActivities/Project_Activity1.py
from selenium import webdriver
from selenium.webdriver.common.by import By

def Project_Activity1():
    driver = webdriver.Chrome()
    driver.get("https://alchemy.hguy.co/lms/")
    pageTitle = driver.title
    assert pageTitle == "Alchemy LMS – An LMS Application"
    print(pageTitle)


Project_Activity1()

File: Selenium/Project/ProjectActivity1.java
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.firefox.FirefoxDriver;
import org.testng.Assert;
import org.testng.annotations.Test;

public class ProjectActivity1 extends BaseTest{

    //    public static void main(String[] args) {
    @Test
    public void titleTest() {


        // Print the title of the page
        System.out.println("Home page title: " + driver.getTitle());
        Assert.assertEquals("Alchemy LMS – An LMS Application", driver.getTitle());

    }
}
